app/src/components/menu.ts

```diff
@@ -132,27 +156,32 @@ export function createMenu({
       },
       {
         label: 'Reload',
-        accelerator: 'CmdOrCtrl+R',
```

Comment:
Did you lose the accelerator on this one?

Reply:
Nope. That's the nice thing about `role`, it provides he accelerator and the function for you for common menu functions:

![Screen Shot 2021-06-04 at 10 37 58 AM](https://user-images.githubusercontent.com/547567/120818934-101f2000-c521-11eb-9da0-df99986d996a.png)